Packs/OTRS/Integrations/OTRS/OTRS.py

```diff
@@ -3,7 +3,7 @@
 import urllib3
 from CommonServerPython import *
```

Comment:
Just my two cents, but maybe move this to the top. When it gets stripped out, it will leave an odd gap in the import statements.